scripts/analyze

```diff
@@ -305,19 +305,22 @@ def write_jrsconfig(list):
                 feature = feature.replace(i, "#" + i)
             p.write(feature)
 
-def write_makefile(src, zconf, file):
-    """Write out a src Makefile"""
-    with open(args['MAKEBASE'], "r") as b:
+def write_cmakefile(src, zconf, file):
+    """Write out a src CMake file"""
+    with open(args['CMAKEBASE'], "r") as b:
         base = b.read()
     with open(file, "w") as f:
-        f.write(base)
```

Comment:
This doesn't seem right, you're no longer using base variable you just made at all.